### PR TITLE
LMB-523 | Set mandataris status to bekrachtigd when besluit is linked

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

We want to automatically update the status of the mandataris when it is found in a besluit.

## How to test

Run the query in your virtuoso endpoint for updating a status of a mandataris.

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/174 => **review this first**

### CODE REVIEW

File: data-access/mandataris.ts [hunk 1]
```diff
@@ -360,3 +359,47 @@ export async function addLinkToDecisionDocumentToMandataris(
     );
   }
 }
+
+export async function updateStatusOfMandataris(
```
Comment (bfidlers) [inline data-access/mandataris.ts:363]: Be careful here, we aren't actually updating the status of the mandataris, we are updating the publication status of the mandataris. 
So I would suggest updating the name of this method. And the predicate used in the query should be `extlmb:hasPublicationStatus`

File: util/constants.ts [hunk 2]
```diff
@@ -2,4 +2,5 @@ export enum MANDATARIS_STATUS {
   BEEINDIGD = 'http://data.vlaanderen.be/id/concept/MandatarisStatusCode/b8866fa2-d61c-4e3d-afaf-8a29eaaa7fb9',
   DRAFT = 'http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/588ce330-4abb-4448-9776-a17d9305df07',
   EFFECTIEF = 'http://data.vlaanderen.be/id/concept/MandatarisStatusCode/21063a5b-912c-4241-841c-cc7fb3c73e75',
+  BEKRACHTIGD = 'http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/9d8fd14d-95d0-4f5e-b3a5-a56a126227b6',
```
Comment (bfidlers) [inline util/constants.ts:5]: Since this is not an actual mandataris status but a mandataris publication status, I would separate these.

**Review by bfidlers — CHANGES_REQUESTED** [review]
Some pretty important issues with the correct relation that should be fixed.

File: util/constants.ts (hunk 2 above)
Comment (bfidlers) [reply util/constants.ts:5]: draft, effectief and bekrachtigd are publication statuses, beeindigd is an actual mandataris status

Comment (JonasVanHoof) [reply util/constants.ts]: Split up the status into mandataris status and publication status

File: util/constants.ts [hunk 3]
```diff
@@ -1,6 +1,9 @@
 export enum MANDATARIS_STATUS {
   BEEINDIGD = 'http://data.vlaanderen.be/id/concept/MandatarisStatusCode/b8866fa2-d61c-4e3d-afaf-8a29eaaa7fb9',
-  DRAFT = 'http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/588ce330-4abb-4448-9776-a17d9305df07',
   EFFECTIEF = 'http://data.vlaanderen.be/id/concept/MandatarisStatusCode/21063a5b-912c-4241-841c-cc7fb3c73e75',
-  BEKRACHTIGD = 'http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/9d8fd14d-95d0-4f5e-b3a5-a56a126227b6',
+}
+
+export enum PUBLICATION_STATUS {
+  DRAFT = 'http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/588ce330-4abb-4448-9776-a17d9305df07',
+  BEKRACHTIGT = 'http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/9d8fd14d-95d0-4f5e-b3a5-a56a126227b6',
```
Comment (bfidlers) [inline util/constants.ts:8]: Zou hier precies wel bekrachtigd zetten van is bekrachtigd niet?

**Review by bfidlers — APPROVED** [review]
Much beter!

**Pull request merged**